libcst/tests/test_batched_visitor.py

```diff
@@ -16,57 +16,57 @@ def test_simple(self) -> None:
         mock = Mock()
 
         class ABatchable(BatchableCSTVisitor):
-            def visit_Pass(self, node: cst.Pass) -> None:
-                mock.visited_a()
-                object.__setattr__(node, "a_attr", True)
+            def visit_Del(self, node: cst.Del) -> None:
+                object.__setattr__(node, "target", mock.visited_a())
 
         class BBatchable(BatchableCSTVisitor):
-            def visit_Pass(self, node: cst.Pass) -> None:
-                mock.visited_b()
-                object.__setattr__(node, "b_attr", 1)
+            def visit_Del(self, node: cst.Del) -> None:
+                object.__setattr__(node, "semicolon", mock.visited_b())
 
-        module = visit_batched(parse_module("pass"), [ABatchable(), BBatchable()])
-        pass_ = cast(cst.SimpleStatementLine, module.body[0]).body[0]
-
-        # Check properties were set
-        self.assertEqual(object.__getattribute__(pass_, "a_attr"), True)
-        self.assertEqual(object.__getattribute__(pass_, "b_attr"), 1)
+        module = visit_batched(parse_module("del 5"), [ABatchable(), BBatchable()])
+        del_ = cast(cst.SimpleStatementLine, module.body[0]).body[0]
 
         # Check that each visitor was only called once
         mock.visited_a.assert_called_once()
         mock.visited_b.assert_called_once()
 
+        # Check properties were set
+        self.assertEqual(object.__getattribute__(del_, "target"), mock.visited_a())
+        self.assertEqual(object.__getattribute__(del_, "semicolon"), mock.visited_b())
+
     def test_all_visits(self) -> None:
         mock = Mock()
 
         class Batchable(BatchableCSTVisitor):
-            def visit_Pass(self, node: cst.Pass) -> None:
-                mock.visit_Pass()
-                object.__setattr__(node, "visit_Pass", True)
+            def visit_If(self, node: cst.If) -> None:
```

Comment:
I had to change to using `If`, because `Pass` didn't have four fields available to reproduce the original test.

due to the slots, arbitrary fields can no longer be set. Existing fields need to be (ab)used.